packages/donor/src/components/AppointmentPicker/AppointmentPicker.tsx

```diff
@@ -16,12 +16,14 @@ export interface AppointmentPickerProps {
   donationDay: DonationDay;
   onSlotSelected: (donationSlot: DonationSlotToBook) => void;
   showHospitalName: boolean;
+  isSwapAppointment: boolean;
```

Comment:
Name is not clear.
What does this field mean? What is it used for? What effect does it have?

Reply:
Clarified and cut unnecessary instances of this variable name.